### ISSUE TITLE
Update to the next release of docker-api when  it has podman support

### ISSUE THREAD

**Issue by trevor-vaughan** [voxpupuli/beaker-docker#35]
Currently, the tests are pinned to my personal fork of swipely/docker-api to provide podman support.

When swipely/docker-api#569 is through and released, then this should be updated appropriately.

**Comment by ekohl** [comment]
Would it be a good idea to fork docker-api to voxpupuli and point the tests there?

**Comment by bastelfreak** [comment]
the PR from @trevor-vaughan was merged and released in docker-api 2.1.0 and 2.2.0. Our gemspec already allows those versions

```
$ bundle info docker-api
  * docker-api (2.2.0)
	Summary: A simple REST client for the Docker Remote API
	Homepage: https://github.com/swipely/docker-api
	Path: /home/bastelfreak/code/beaker-docker/.vendor/ruby/3.0.0/gems/docker-api-2.2.0
```
